3rdparty/amd/tuning/benchmark_moe_rocm.py

```diff
@@ -0,0 +1,380 @@
+import argparse
+import json
+import os
+import sys
+
+import torch
+import torch.nn.functional as F
+import triton
+import triton.language as tl
+from tqdm import tqdm
+from vllm import _custom_ops as ops
+
+from sglang.srt.layers.fused_moe.fused_moe import fused_moe, get_config_file_name
+
+padding_size = 128 if bool(int(os.getenv("MOE_PADDING", "0"))) else 0
+
+
+def main(model, tp_size, gpu, dtype: str, batches):
+    method = fused_moe
+
+    for bs in batches:
+        run_grid(
+            int(bs), model=model, method=method, gpu=gpu, tp_size=tp_size, dtype=dtype
+        )
+
+
+def prune_configs(M, N, K, configs):
+    pruned_configs = []
+    elemBytes_a = 1  # [DV Note] Hard-coded for float16 (2 bytes)
+    elemBytes_b = 1  # [DV Note] Hard-coded for float16 (2 bytes)
+
+    mfma = 16 if M < 32 or N < 32 else 32
```

Comment:
`4x4` when it is supported?

Reply:
The Triton team does not give the specific date for 4x4 support